src/compojure/api/meta.clj

```diff
@@ -607,7 +635,8 @@
                           :body body}
                          options)
 
-        static? (not (or dynamic? (route-args? route-arg) (seq lets) (seq letks)))
+        static? (not (or (-> info :public :dynamic boolean)
```

Comment:
Why boolean?

Reply:
It used to be `dynamic?` which indicated a `boolean`. fixed.